instrumentation/opentelemetry-instrumentation-aiopg/tests/test_aiopg_integration.py

```diff
@@ -199,7 +199,11 @@ def test_span_succeeded(self):
             "user": "user",
         }
         db_integration = AiopgIntegration(
-            self.tracer, "testcomponent", "testtype", connection_attributes
+            self.tracer,
```

Comment:
Aren't there a lot of other instrumentations that use dbapi as well (mysql, psycopg2, etc)? I think tests would need to be added for those as well because this is a breaking change.

Reply:
I would argue that since the the other instrumentations already are not testing for "db.statement.parameters", we would not need to modify the test to do so now.

Also db.statement.parameters is not even part of the specification: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes, why are we collecting it at all?

Reply:
> I would argue that since the the other instrumentations already are not testing for "db.statement.parameters", we would not need to modify the test to do so now.

To ensure completeness of the feature, tests should be added for each instrumentation. The original authors should have tested for this, but since your change will effect the behavior of these instrumentations, it would be good to have them there as well.

> why is it being collected at all?

That is a good question. Perhaps this should be the discussion topic to address first? @codeboten